GitExtensionsVSIX/Git/GitCommands.cs

```diff
@@ -25,7 +25,7 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             }
             catch (UnauthorizedAccessException)
             {
-                MessageBox.Show("Git Extensions has insufficient permissions to check the registry.");
+                MessageBox.Show("Git Extensions has insufficient permissions to check the registry.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
There is a "Error" string in MessageBoxes.cs
You should probably add the string to Resources instead

Reply:
The main message is not translated so I don't think it's necessary. But what should I do for translated message ? Had a "Error" translated ? Or add a more specific message like "Insufficient permissions" for this example ?

Reply:
@gerhardol this is a VSIX project, we don't have translations for it nor mechanism to switch languages

Reply:
This was a general comment for all - I missed that this specific was vsix. But RussKie also commented on that.